### PR TITLE
Connect form submit to database & add display pages for forms

### PR DESCRIPTION
**Pull request by fennifith**
This connects the forms "submit" buttons to the `insertForm` API route, and redirects the user to the page of the created content node upon submitting.

Additionally, if the form has errors when "submit" is clicked, the page will focus & scroll to the erroring element (provided it has an `id` that exactly matches the field name).

### CODE REVIEW

File: src/components/info/content/Institution.svelte [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	$: names = content.name.split(",");
+	$: json = content.content as any;
+</script>
+
+<div class="mb-5">
+	<h1>{names[0]}</h1>
+
+	{#if names.length > 1}
+	<small>
+		<h6>Also known as:</h6>
```
Comment (benmiller314) [inline src/components/info/content/Institution.svelte]: For accessibility reasons, we shouldn't jump from h1 to h6: if this is the next level of heading, let's call it h2 and resize h2 within this scope (e.g. `small h2`) to the desired size. Alternately, we could replace `<small>` with something more semantic like `<header>` and just have the "Also known as:" text be `<strong>` within it. Or the combination: `<header><h2>`.

File: src/components/info/content/Institution.svelte [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		<i class="material-icons me-2">place</i>
+		{json.location}
+	</div>
+	{/if}
+</div>
+
+<div class="row">
+	<div class="col col-12 col-lg-8">
+		{#if json.description}
+		<h5>Description</h5>
```
Comment (benmiller314) [inline src/components/info/content/Institution.svelte]: Likewise, Description seems to be actually at the h2 level within the page.

File: src/components/info/content/Institution.svelte [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		<p>{json.description}</p>
+		{/if}
+
+		{#if json.tags && json.tags.length > 0}
+		<div class="mb-3">
+			{#each json.tags as tag}
+			<span class="badge rounded-pill bg-secondary me-2">{tag}</span>
```
Comment (benmiller314) [inline src/components/info/content/Institution.svelte:41]: When we have a view that lists nodes by tag, I'd like each of these `span.badge`s to link to such a list.

File: src/components/info/content/Institution.svelte [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+			{/each}
+		</div>
+		{/if}
+	</div>
+	<div class="col col-12 col-lg-4">
+		{#if json.websites && json.websites.length > 0}
+		<h5>Websites</h5>
```
Comment (benmiller314) [inline src/components/info/content/Institution.svelte]: Just flagging this h5 again. Still h2, I think?

File: src/components/info/content/Person.svelte [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+		<i class="material-icons me-2">volume_up</i>
+		Hear a pronunciation
+	</a>
+	{/if}
+
+	<h1>{names[0]}</h1>
+
+	{#if names.length > 1}
+	<small>
+		<h6>Also known as:</h6>
```
Comment (benmiller314) [inline src/components/info/content/Person.svelte]: See comment above re: `<small>` and `<h6>`: probably more like `h2`

Comment (benmiller314) [reply src/components/info/content/Person.svelte]: I'll stop flagging these...

File: src/components/info/content/Person.svelte [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+	<div class="col col-12 col-lg-8">
+		{#if json.description}
+		<h5>Description</h5>
+		<p>{json.description}</p>
+		{/if}
+
+		{#if json.tags && json.tags.length > 0}
+		<div class="mb-3">
+			{#each json.tags as tag}
+			<span class="badge rounded-pill bg-secondary me-2">{tag}</span>
```
Comment (benmiller314) [inline src/components/info/content/Person.svelte:51]: Again, flagging for later to link these

Comment (benmiller314) [reply src/components/info/content/Person.svelte:51]: etc

File: src/routes/forms/[uuid]/index.svelte [hunk 7]
```diff
@@ -53,7 +73,7 @@
 		{/if}
 	{/each}
 
-	<button class="btn btn-primary indent" on:click={handleSubmit}>Save and continue</button>
+	<button class="btn btn-primary indent" on:click={handleSubmit}>Submit</button>
```
Comment (benmiller314) [inline src/routes/forms/[uuid]/index.svelte]: I'd like to hear more about this – I'd really liked the idea of "save and continue." Is that not going to work?

**Review by benmiller314 — COMMENTED** [review]
Thanks, Ben and James! My big questions for the morning:

* How close are we to getting relations running?
* What are the blockers on Explore?
* What are the blockers on edit node?

A bunch of repetitive comments below relate to accessibly semantic html (not skipping heading levels, mostly). Also putting in a reminder that we should make the tags clickable when it's possible to explore by tag.

File: src/routes/forms/[uuid]/index.svelte (hunk 7 above)
Comment (fennifith) [reply src/routes/forms/[uuid]/index.svelte]: My assumption was that the "save and continue" button should enter a user-side review state without inserting any nodes into the database, and that the user would have one final "confirm/submit" button in that review state to finally process the form (which is what I intended this button to be) - did I misunderstand the idea behind that?

Comment (benmiller314) [reply src/routes/forms/[uuid]/index.svelte]:
```suggestion
	<button class="btn btn-primary indent" on:click={handleSubmit}>Continue</button>
```

**Pull request merged**